test/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -68,6 +69,7 @@ test_cases() ->
      session_can_be_kicked,
      messages_are_sent_and_received,
      messages_are_archived,
+     archive_and_paginate,
```

Comment:
Probably function name `messages_can_be_paginated` describes better what the test cases checks.